templates/yahoo_answers_qa/templates.yaml

```diff
@@ -0,0 +1,45 @@
+dataset: yahoo_answers_qa
+templates:
+  1a6eda86-debc-4681-b643-f1f16fedd713: !Template
+    id: 1a6eda86-debc-4681-b643-f1f16fedd713
+    jinja: "{% if nbestanswers|length > 1 %} \n\nGive the top two answers for the\
+      \ following question:\n\n{{question}} |||\n{{nbestanswers[0]}}\n\n{{nbestanswers[1]}}\n\
+      \n{% endif %}"
+    name: n_best_answer
+    reference: ''
+  2c214261-f32b-42ab-ac90-b22f4f2f465f: !Template
+    id: 2c214261-f32b-42ab-ac90-b22f4f2f465f
+    jinja: 'What is the category of the following question?
+
+
+      {{question}} |||
+
+      {{main_category}}
+
+      '
+    name: category
+    reference: ''
+  7ba7a99c-be14-47d4-859d-093ce07a2798: !Template
+    id: 7ba7a99c-be14-47d4-859d-093ce07a2798
+    jinja: 'Given the n-best answers , {{nbestanswers}}, generate the question |||
```

Comment:
This one doesn't compile. I think you need to join the list. Also the phrase "n-best" is too technical. Can you say the "top answers"